radio/src/lua/api_general.cpp

```diff
@@ -1535,7 +1535,20 @@ static int luaGetUsage(lua_State * L)
   lua_pushinteger(L, instructionsPercent);
   return 1;
 }
+/*luadoc
+@function getMemory()
```

Comment:
I would name it getAvailableMemory()

Reply:
> Could you explain its intended use?just confused.
Its just a hookup for LUA to the same function call by OTX that shows the bytes remaining on the debug statistics page, that way, any LUA scripts created by users can take into account mem remaining on the radio (heap) to prevent itself from being killed (by staging scripts) or for debugging without needed to exit the telemetry page (If running custom Telemetry Screen).